11-Backbonejs/collections.md

```diff
@@ -0,0 +1,164 @@
+# Organizing Data with Collections
+
+As we've worked with JavaScript, up until this point all our data in has been stored as arrays of objects. This is fine for simple programs, but for more complex applications we'll need a more robust way to organize things. In the Backbone framework, this problem is solved through Models and Collections.
```

Comment:
This paragraph is the exact same text we have in `models.md`, is that intentional?

If we want to keep it we should fix "all our data in" -> "all our data".